src/System.Net.Sockets/ref/System.Net.Sockets.csproj

```diff
@@ -1,22 +1,17 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <ProjectGuid>{834E3534-6A11-4A8D-923F-35C1E71CCEC3}</ProjectGuid>
-    <IsPartialFacadeAssembly Condition="'$(TargetGroup)' == 'netfx'">true</IsPartialFacadeAssembly>
     <Configurations>netcoreapp-Debug;netcoreapp-Release;uap-Debug;uap-Release</Configurations>
   </PropertyGroup>
   <ItemGroup>
     <Compile Include="System.Net.Sockets.cs" />
-  </ItemGroup>
-  <ItemGroup Condition="'$(TargetGroup)' != 'netfx'">
     <Compile Include="System.Net.Sockets.netcoreapp.cs" />
+  </ItemGroup>
```

Comment:
NIT: Given that `System.Net.Sockets.netcoreapp.cs` will be included every time, perhaps we should just fold the contents into `System.Net.Sockets.cs`

Reply:
Sounds nice, but just a little bit more involved than I wanted this to be, so hope its OK if I pass on that one!